python/dgl/distributed/sparse_emb.py

```diff
@@ -3,9 +3,11 @@
 from .. import backend as F
 from .. import utils
 from .dist_tensor import DistTensor
+import torch.distributed as dist
+import torch as th
 
-class DistEmbedding:
-    '''Distributed embeddings.
+class NodeEmbedding:
```

Comment:
For compatibility, we should still expose DistEmbedding. We can deprecate the name and remove it in the next release.

Reply:
You mean we keep the entire DistEmbedding implementation?